content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -46,73 +47,61 @@ monitoring statistics by default:
 
 ### Kubelet
 
-The Kubelet acts as a bridge between the Kubernetes master and the nodes. It manages the pods and containers running on a machine. Kubelet translates each pod into its constituent containers and fetches individual container usage statistics from the container runtime, through the container runtime interface. For the legacy docker integration, it fetches this information from cAdvisor.  It then exposes the aggregated pod resource usage statistics through the kubelet resource metrics api.  This api is served at `/metrics/resource/v1alpha1` on the kubelet's authenticated and read-only ports.
+The Kubelet acts as a bridge between the Kubernetes master and the nodes. 
+It manages the pods and containers running on a machine. Kubelet translates each pod 
```

Comment:
s/Kubelet/The kubelet

See https://github.com/kubernetes/website/pull/14831#issuecomment-518871961 for context.

Reply:
updated